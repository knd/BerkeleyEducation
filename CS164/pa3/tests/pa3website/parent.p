parent(john, ed).
parent(bob, john).
gp(X,Y) :- parent(X,Z), parent(Z,Y).

gp(W, ed)?
