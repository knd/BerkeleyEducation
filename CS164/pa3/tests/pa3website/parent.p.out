Asking for solution 1
W = bob
Asking for solution 2
None
